presto-main/src/test/java/com/facebook/presto/operator/scalar/sql/TestArrayArithmeticFunctions.java

```diff
@@ -65,4 +74,41 @@ public void testArrayAverage()
         assertFunction("array_average(array[null, null])", DOUBLE, null);
         assertFunction("array_average(null)", DOUBLE, null);
     }
+
+    @Test
+    public void testArrayFrequencyBigint()
```

Comment:
I think we should also add tests for the empty array case, and for nulls in the array.

Reply:
Let's also add test for empty array as @prithvip suggested. `array_frequency(cast(array[] as array(bigint)`. Also, for bigint / varchar, either use all upper case or all lower case. SQL is case in sensitive, I think here you can use all lower case.